builds/cmake/gen-cmake-presets.py

```diff
@@ -0,0 +1,103 @@
+#!/bin/env python3
+
+# Script to generate all combinations for the CMakePresets.json
+# Edit the CMakePresets.json.template and run this script
+# This script and all json files under CC0
```

Comment:
CC0 for code is scary for some distros, the Unlicense fits better because there are no patent-related issues with it. See https://github.com/20kdc/gabien-app-r48/issues/56